Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -348,6 +358,16 @@ open class NavigationMapView: UIView {
         mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
             guard let self = self else { return }
             self.pointAnnotationManager = self.mapView.annotations.makePointAnnotationManager()
+            
+            if let finalDestinationAnnotation = self.finalDestinationAnnotation,
+               let pointAnnotationManager = self.pointAnnotationManager {
+                pointAnnotationManager.syncAnnotations([finalDestinationAnnotation])
```

Comment:
Just curious, is it possible the SDK user adds it's own annotations to `pointAnnotationManager ` and we will rewrite them here? Maybe it makes sense to merge what `pointAnnotationManager.annotations` has and add a final destination annotation?

Reply:
Thanks for clarifications